apps/main/src/app/Home/Home.tsx

```diff
@@ -1,375 +1,121 @@
-              your <strong>.env</strong> file.
-            </p>
-          </div>
-        </div>
-      </section>
-
-      {/* Project Structure Section */}
-      <section className="mb-8">
-        <h2 className="text-2xl font-bold mb-4">Project Structure</h2>
-        <p className="mb-4">
-          The project is organized as an Nx Monorepo with the following structure:
-        </p>
-        <pre className="bg-gray-100 p-4 rounded-lg">
-          <code>
-            {`remr-stack-template/
-├── apps/
-│   └── main/                # React application
-├── libs/                    # Shared libraries (e.g., utilities, types)
-├── backend/
-│   ├── src/                 # Backend files
-│   │   └── server.js        # Backend entrypoint
-│   └── package.json         # Backend package.json
-├── dist/                    # Builds and other outputs
-├── docker-compose.yml       # Docker Compose configuration
-├── docker-compose.prod.yml  # Docker Compose configuration for production
-├── .env                     # Environment variables
-├── nx.json                  # Nx workspace configuration
-├── package.json             # Root package.json
-└── README.md                # This file`}
-          </code>
-        </pre>
-      </section>
-
-      {/* Available Scripts Section */}
-      <section className="mb-8">
-        <h2 className="text-2xl font-bold mb-4">Available Scripts</h2>
-        <ul className="list-disc list-inside space-y-2">
-          <li><code>start:main</code>: Starts the frontend in development mode (without Docker).</li>
-          <li><code>build:main</code>: Builds the frontend for production.</li>
-          <li><code>test:main</code>: Runs tests for frontend apps.</li>
-          <li><code>lint:main</code>: Lints the monorepo project defined in scripts.</li>
-          <li><code>e2e:main</code>: Runs e2e tests for monorepo project defined in scripts.</li>
-          <li><code>start:backend</code>: Starts the Express server.</li>
-        </ul>
-      </section>
+      {/* Glavna vsebina */}
+      <div className="flex-1 ml-64">{renderContent()}</div>
+    </div>
+  );
+}
 
-      {/* Docker Configuration Section */}
-      <section className="mb-8">
-        <h2 className="text-2xl font-bold mb-4">Docker Configuration</h2>
-        <p className="mb-4">
-          The <code>docker-compose.yml</code> file defines the following services:
-        </p>
-        <ul className="list-disc list-inside space-y-2">
-          <li><strong>backend</strong>: Express app served on port 8000.</li>
-          <li><strong>mariadb</strong>: MariaDB database.</li>
-          <li><strong>redis</strong>: Redis server for caching.</li>
-        </ul>
-        <p className="mt-4">
-          You can customize the Docker configuration by editing the <code>docker-compose.yml</code> file or the
-          individual Dockerfiles.
-        </p>
-      </section>
+// Komponenta Sidebar (meni)
+function Sidebar({ activeTab, setActiveTab }: any) {

```

Comment:
```suggestion
const Sidebar = ({ activeTab, setActiveTab }: any) => {
```

pa komponente buls da das v druge file, ce so globalne komponente (te k bodo tut na drugih appih) jih das pa v libs